deploy/kubernetes/examples/demo-pvc-clone-pvc.yaml

```diff
@@ -0,0 +1,15 @@
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: demo-pvc-cloned-pvc
+spec:
+  volumeMode: Filesystem
```

Comment:
What about option for block (raw)?

Reply:
I added to the line

Reply:
for snapshots, we didn't add a yaml for raw block.
so do you think we are missing two yamls?
IMO the examples should not cover every possible feature combination, and each feature is covered in at east 1 example

Reply:
@ArbelNathan if you think it should be added to the line (which might complicate things), then it should also be added in `demo-pvc-from-snapshot.yaml`, and possibly in the `dataSource: name:` lines

Reply:
well, its "example" not a UG.. I can just remove the block comment and leave it as it was..

Reply:
I think removing the comment would be best

Reply:
done

Reply:
@roysahar-ibm since you opened this conversation, please resolve it once you think it is resolved

Reply:
We can think on all the yamls that maybe missing but since the examples pointed to tech people that have k8s knowledge we can have the minimum examples.